bentoml/handlers/dataframe_handler.py

```diff
@@ -67,10 +156,14 @@ class DataframeHandler(BentoHandler):
     """
 
     def __init__(
-        self, orient="records", output_orient="records", typ="frame", input_dtypes=None
+        self, orient="records", output_orient="records", typ="frame", input_dtypes=None,
+        micro_batch=False, mb_max_latency=0.8,
```

Comment:
Do we really need the `micro_batch` parameter here?

It seems to me, whether for `micro_batch` to be True or False, it is the same for the user-defined API callback function(in case of Dataframe handler where input is already a 'batch').  I think ideally, we want users to just define the callback function to process a batch of inputs so that BentoML can load the BentoService to run with micro-batch enabled or disabled.

Reply:
Instead, we could add a constant `batch_mode_supported` to BentoHandler and default to false, that way on the marshal server-side, we can determine if a BentoServiceAPI endpoint should be loaded for batch process or not. This can be useful when we are rolling out this feature as some of the handlers does not yet support it.